src/Sulu/Bundle/PreviewBundle/Resources/js/views/Preview/Preview.js

```diff
@@ -0,0 +1,265 @@
+// @flow
```

Comment:
Is it a good idea to put that into the `view` folder? `Views` are normally components that have the `ViewProps` type, and are rendered into the main part of the application. Maybe we should introduce another folder for sidebar-views? or just leave it as a container? To be discussed.

Reply:
@chirimoya

Reply:
`container`

Reply:
`SidebarViewRegistry` => `SidebarRegistry`